bioimageio_core_version: 0.5.8
bioimageio_spec_version: 0.4.9
rdf_sha256: eb5321a89f09e1596437caab4c1308931a5a71ac746e38e018e169d000244e98
status: failed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings:
      tags: 'Missing tags corresponding to bioimage.io categories: [{''modality'':
        [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''dims'': [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''content'': [''cells'',
        ''nuclei'', ''extracellular-vesicles'', ''tissue'', ''plant'', ''mitochondria'',
        ''vasculature'', ''cell-membrane'', ''brain'', ''whole-organism'']}, {''framework'':
        [''tensorflow'', ''pytorch'', ''tensorflow.js'']}, {''software'': [''ilastik'',
        ''imagej'', ''fiji'', ''imjoy'', ''deepimagej'', ''n2v'']}, {''network'':
        [''unet'', ''densenet'', ''resnet'', ''inception'', ''shufflenet'']}, {''task'':
        [''semantic-segmentation'', ''instance-segmentation'', ''object-detection'',
        ''image-classification'', ''denoising'', ''image-restoration'', ''image-reconstruction'',
        ''in-silico-labeling'']}]'
      weights: {torchscript: missing 'pytorch_version'}
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9
    nested_errors: {}
    status: passed
    traceback: null
    warnings:
      tags: 'Missing tags corresponding to bioimage.io categories: [{''modality'':
        [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''dims'': [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''content'': [''cells'',
        ''nuclei'', ''extracellular-vesicles'', ''tissue'', ''plant'', ''mitochondria'',
        ''vasculature'', ''cell-membrane'', ''brain'', ''whole-organism'']}, {''framework'':
        [''tensorflow'', ''pytorch'', ''tensorflow.js'']}, {''software'': [''ilastik'',
        ''imagej'', ''fiji'', ''imjoy'', ''deepimagej'', ''n2v'']}, {''network'':
        [''unet'', ''densenet'', ''resnet'', ''inception'', ''shufflenet'']}, {''task'':
        [''semantic-segmentation'', ''instance-segmentation'', ''object-detection'',
        ''image-classification'', ''denoising'', ''image-restoration'', ''image-reconstruction'',
        ''in-silico-labeling'']}]'
      weights: {torchscript: missing 'pytorch_version'}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: load resource description
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: https://bioimage-io.github.io/collection-bioimage-io/rdfs/10.5281/zenodo.6200999/7678300/rdf.yaml
    name: All URLs and paths available
    nested_errors: null
    status: failed
    traceback: ["  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/core/resource_tests.py\",
        line 87, in _test_resource_urls\n    SourceNodeChecker(root_path=rd.root_path).visit(rd)\n",
      "  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 34, in visit\n    visitor(node)\n", "  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/core/resource_io/utils.py\",
        line 52, in generic_visit\n    self.visit(value)\n", "  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 34, in visit\n    visitor(node)\n", "  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/core/resource_io/utils.py\",
        line 38, in visit_URI\n    self._visit_source(node)\n", "  File \"/home/runner/micromamba-root/envs/7678300/lib/python3.9/site-packages/bioimageio/core/resource_io/utils.py\",
        line 35, in _visit_source\n    raise FileNotFoundError(leaf)\n"]
    warnings: {}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: Test documentation completeness.
    status: passed
    traceback: null
    warnings: {documentation: No '# Validation' (sub)section found.}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: reproduce test outputs from test inputs
    status: passed
    traceback: null
    warnings:
      non-validation-warnings: [Implicit dimension choice for softmax has been deprecated.
          Change the call to include dim=X as an argument.]
  deepimagej:
  - {bioimageio_spec_version: 0.4.8post1, error: Sample image tiffs have not been
      generated from numpy tests files, name: Initial compatibility checks with DeepImageJ,
    status: failed}
  ilastik:
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: local variable 'rd' referenced before assignment
    name: reproduce test outputs with ilastik 1.4.0rc8 using torchscript weights
    nested_errors: null
    status: failed
    traceback: ["  File \"/home/runner/work/bioimage-io-resources/bioimage-io-resources/scripts/test_with_ilastik.py\",
        line 82, in run_tests\n    summaries = test_model(rdf_path, weight_format=weight_format)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 41, in test_model\n    model_rdf, weight_format=weight_format, devices=devices,
        decimal=decimal, expected_type=\"model\"\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 250, in test_resource\n    rd, load_test = _test_load_resource(rdf, weight_format)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 224, in _test_load_resource\n    return rd, load_summary\n"]
    warnings: {}
  - {error: null, name: has expected resource type, status: passed, traceback: null}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: All URLs and paths available
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
